includes/Products.php

```diff
@@ -497,8 +497,107 @@ public static function update_commerce_enabled_for_product( \WC_Product $product
 	 */
 	public static function get_google_product_category_id( \WC_Product $product ) {
 
-		// TODO: implement
-		return '';
+		// attempt to get from product or parent product metadata
+		if ( $product->is_type( 'variation' ) ) {
+			$google_product_category_id = get_post_meta( $product->get_parent_id(), self::GOOGLE_PRODUCT_CATEGORY_META_KEY, true );
```

Comment:
we need to implement the same update here to use `\WC_Product::get_meta()` instead, after checking for an instance